### PR TITLE
Class format

### PR DESCRIPTION
**Pull request by fobadara**
## Awesome Books: Code Refactorisation

In this branch:

🔴  We refactored the code with Object-Oriented Programming.

### CODE REVIEW

File: javascript/index.js [hunk 1]
```diff
@@ -2,60 +2,69 @@ let books = JSON.parse(localStorage.getItem('books'));
...
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    const book = { title, author };
+    storedBooks = storedBooks.filter((book) => book.title !== title || book.author !== author);
+    storedBooks.push(book);
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(storedBooks));
+    storedBooks = JSON.parse(localStorage.getItem('books'));
+    window.location.reload();
+  }
```
Comment (Emmyn5600) [inline javascript/index.js]: - [ ] Nice to use array methods that doesn't mutate the array, however, Let us prevent to enter empty data, kindly validate the input fields.
cacl3

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @fobadara and @Kingstalux 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:

Changes required :writing_hand: 
 
  - Please check the comment under the review.

  Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: javascript/index.js (hunk 1 above)
Comment (fobadara) [reply javascript/index.js]: Hello Emmyn, THanks for the review. We were already working on that in the next feature branch. but still, I believe this project should have been approved as it is not a requirement but an optional suggestion.

**Review by descholar-ceo — APPROVED** [review]
Hi @fobadara and @Kingstalux , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**